### PR TITLE
Fixed behavior when selinux is disabled

### PR DESCRIPTION
**Pull request by japokorn**
selinux flag is set to True for the duration of the test

### CODE REVIEW

**Comment by dwlehman** [comment]
This test should be performing the (fake) mount both with `flags.selinux` enabled and disabled to verify that the flag correctly controls whether we try to adjust the context on the mountpoint and `lost+found` directory.

**Comment by japokorn** [comment]
I have moved tearDown to _clean_up. PR should be ready for closing now.

**Comment by dwlehman** [comment]
Please rebase so there is no merge commit on the branch. Other than that, it looks good.

File: tests/formats_test/selinux_test.py [hunk 1]
```diff
@@ -51,3 +52,8 @@ def test_mount_selinux_ext2fs(self):
     def test_mount_selinux_xfs(self):
         """ Test of correct selinux context parameter value when mounting XFS"""
         self.exec_mount_selinux_format(fs.XFS)
+
+    def _clean_up(self):
+        super(SELinuxContextTestCase, self).tearDown()
```
Comment (vojtechtrefny) [inline tests/formats_test/selinux_test.py]: Please remove this line. We don't want to call `tearDown` from `_clean_up` method.

**Comment by japokorn** [comment]
Commits squashed

**Comment by japokorn** [comment]
pushed

**Pull request merged**